glossary/general/comment.md

```diff
@@ -0,0 +1,3 @@
+# Comment
+
+Comments are used to explain the code. They do not have any effect on the execution and they are written using #. For instance, # This is a comment.
```

Comment:
Should we show code output here, such that comments are colored?

Reply:
Yes, we should. I missed that while copying, great catch.

Reply:
how about

``````suggestion
Comments are used to explain the code. They do not have any effect on the execution. Depending on the programming language, there are multiple ways to create a *comment*:

```py
# I am a Python comment!
```

```js
// I am a JavaScript comment!
```

```html
<!-- I am an HTML comment! -->
```
``````

Reply:
we should also update the copy to highlight the importance of comments (especially [Code Tells You How, Comments Tell You Why](https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/))